src/components/input-time/input-time.e2e.ts

```diff
@@ -0,0 +1,360 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { localizeTimeString, localizeTimeStringToParts } from "../../utils/time";
+import {
+  accessible,
+  defaults,
+  disabled,
+  focusable,
+  formAssociated,
+  labelable,
+  reflects,
+  renders
+} from "../../tests/commonTests";
+import { CSS } from "./resources";
+
+describe("calcite-input-time", () => {
+  it("renders", async () => renders("calcite-input-time", { display: "inline-block" }));
+
+  it("is accessible", async () =>
+    accessible(`
+    <calcite-label>
+      Input Time
+      <calcite-input-time name="test"></calcite-input-time>
+    </calcite-label>
+  `));
+
+  it("has defaults", async () =>
+    defaults("calcite-input-time", [
+      { propertyName: "scale", defaultValue: "m" },
```

Comment:
does it make sense to standardize around storing defaults in `./resources.ts` so they can be referenced in the component and tests?

Reply:
Yeah I like that idea